File: bookshop-java/src/main/java/org/bookshop/cart/cartItem/infrastructure/CartItemWriteModel.java
package org.bookshop.cart.cartItem.infrastructure;

public class CartItemWriteModel {
    private String cartId;
    private String productId;
    private int quantity;

    public CartItemWriteModel(){

    }

    public CartItemWriteModel(String cartId, String productId, int quantity) {
        this.cartId = cartId;
        this.productId = productId;
        this.quantity = quantity;
    }

    public String getCartId() {
        return cartId;
    }

    public String getProductId() {
        return productId;
    }

    public int getQuantity() {
        return quantity;
    }

    public void setCartId(String cartId) {
        this.cartId = cartId;
    }

    public void setProductId(String productId) {
        this.productId = productId;
    }

    public void setQuantity(int quantity) {
        this.quantity = quantity;
    }
}

File: bookshop-angular/src/app/cart/cartItem/cartItemWriteModel.ts
export class CartItemWriteModel{
    private cartId:string;
    private productId: string;
    private quantity: number;
    constructor(cartId: string, productId: string, quantity: number){
        this.cartId = cartId;
        this.productId = productId;
        this.quantity = quantity;
    }
}